pom.xml

```diff
@@ -62,7 +62,7 @@
         </pluginRepository>
     </pluginRepositories>
     <properties>
-        <jenkins.version>1.642.3</jenkins.version>
+        <jenkins.version>2.7.3</jenkins.version>
```

Comment:
To test against `workflow-cps` 2.33. No longer strictly necessary since we are no longer relying on `GraphListener` as an extension point, but it is high time we moved to Jenkins 2 anyway right?